mmv1/products/artifactregistry/Repository.yaml

```diff
@@ -27,7 +27,6 @@ update_verb: 'PATCH'
 update_mask: true
 import_format:
   - 'projects/{{project}}/locations/{{location}}/repositories/{{repository_id}}'
-  - '{{repository_id}}'
```

Comment:
note: I originally had `{{location}}/{{repository_id}}` here explicitly, but from what I could see, what's on line 29 seems to already make this work automagically.